### DISCUSSION TITLE
Raw Keyboard Not Working

### DISCUSSION THREAD

**Discussion by sofia-bg** [UNC-Project-Open-AAC/OS-DPI#256] [Help]
@gbishop 
I don't seem to be able to use the keyboard on my computer to change the access method in this layout anymore. 

It should do the following: 
t = headmouse on 
T = headmouse off 
s = setup open

[autoscalable_kybd_061724-1.zip](https://github.com/user-attachments/files/17319958/autoscalable_kybd_061724-1.zip)

**Comment by gbishop** [comment]
Thanks for the report. I have fixed it. It should work again after you update.
